src/index.html

```diff
@@ -6,9 +6,52 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header>
+      <div class="header container">
+        <a class="logo" href="#home">
+          <img src="images/logo.png" alt="MoYo Logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">Apple</a>
            </li>
            
            <li class="nav__item">
              <a href="#samsung" class="nav__link">Samsung</a>
            </li>
            
            <li class="nav__item">
```

Add blank line between multiple sibling blocks of html